app/controllers/application_controller.rb

```diff
@@ -67,6 +69,12 @@ def timeout_timer
 
 private
 
+  def check_service_availability
```

Comment:
Can we consider reworking this a little to use a guard clause, with `dry-types` and set the config on the `Rails.application` for consistency.

Reply:
Add `SERVICE_UNAVAILABLE` to `.env.example` and README.

Reply:
[81ac629](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/941/commits/81ac629e5d06a1919ac6515e7de1b3ed09f759aa)